apiserver/apiserver.go

```diff
@@ -786,13 +778,9 @@ func (srv *Server) trackRequests(handler http.Handler) http.Handler {
 }
 
 func (srv *Server) apiHandler(w http.ResponseWriter, req *http.Request) {
-	atomic.AddInt64(&srv.totalConn, 1)
-	addCount := func(delta int64) {
-		atomic.AddInt64(&srv.connCount, delta)
-	}
-
-	addCount(1)
-	defer addCount(-1)
+	srv.metricsCollector.TotalConnections.Inc()
+	srv.metricsCollector.APIConnections.Inc()
```

Comment:
If this is the api vs. apiserver one, can we add a comment with a note to remove for 2.6?